styleguide/components/11-base/boxes/_boxes.scss

```diff
@@ -94,6 +95,31 @@
         height: 7rem;
       }
     }
+
+    .accolade {
+      bottom: -1rem;
+      background-color: inherit;
+
+      &[class^='top'],
```

Comment:
Die eerste class lijkt me overbodig?

Reply:
Je hebt ze beide nodig voor het geval die klasse de eerste is in de classlist of niet.